.gitignore

```diff
@@ -25,6 +25,7 @@ yarn-debug.log*
 yarn-error.log*
 
 # local env files
+.env
```

Comment:
random thought: should we consolidate on just using `.env` and no `.env.local`?

Reply:
would be great, its difficult to handle this both with drizzle and prisma for some weird reason.